frontend/public/components/_nav-title.scss

```diff
@@ -17,11 +17,15 @@
     padding-left: $grid-gutter-width;
     padding-right: $grid-gutter-width;
   }
-  &--breadcrumbs {
-    padding-top: 0;
-  }
   &--detail {
     border-bottom: 1px solid $color-grey-background-border;
+    // Set min-height to prevent shifting of page content when actions btn is rendered after page content
+    min-height: 90px; // h1 text + top & bottom margin + bottom border (29 + 30 + 30 + 1)
+  }
+  &--breadcrumbs {
```

Comment:
Nit:  alpha

Reply:
Reordered this to enable `--breadcrumbs` `min-height` to take precedence, since it'll be a sibling with `--detail`

Reply:
I'm okay with mis-alphbetization if there is a reason, but add a comment explaining why so anyone else reading this later knows why.